### ISSUE TITLE
Deserialisation error

### ISSUE THREAD

**Issue by pditommaso** [seqeralabs/nf-tower#225]
```
logs/tower.2020-04-05-1.log:Apr-05 00:00:02.307 [nioEventLoopGroup-8-2] ERROR i.s.tower.controller.BaseController - Oops.. something went wrong
logs/tower.2020-04-05-1.log-- request: [PUT] /trace/g81oO0Qo/progress
logs/tower.2020-04-05-1.log-- params : []
logs/tower.2020-04-05-1.log-- user   : [omitted]
logs/tower.2020-04-05-1.log-- message: [io.micronaut.core.convert.exceptions.ConversionErrorException] Failed to convert argument [req] for value [null] due to: Cannot deserialize instance of `java.lang.String` out of START_OBJECT token
logs/tower.2020-04-05-1.log- at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: io.seqera.tower.exchange.trace.TraceProgressRequest["tasks"]->java.util.ArrayList[0]->io.seqera.tower.domain.Task["tag"])
logs/tower.2020-04-05-1.log-
logs/tower.2020-04-05-1.log-io.micronaut.core.convert.exceptions.ConversionErrorException: Failed to convert argument [req] for value [null] due to: Cannot deserialize instance of `java.lang.String` out of START_OBJECT token
logs/tower.2020-04-05-1.log- at [Source: UNKNOWN; line: -1, column: -1] (through reference chain: io.seqera.tower.exchange.trace.TraceProgressRequest["tasks"]->java.util.ArrayList[0]->io.seqera.tower.domain.Task["tag"])
logs/tower.2020-04-05-1.log-	at io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:276)
logs/tower.2020-04-05-1.log-	at io.micronaut.web.router.RouteMatch.execute(RouteMatch.java:122)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.server.netty.RoutingInBoundHandler.lambda$buildResultEmitter$17(RoutingInBoundHandler.java:1400)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableCreate.subscribeActual(FlowableCreate.java:71)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableMap.subscribeActual(FlowableMap.java:37)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchIfEmpty.subscribeActual(FlowableSwitchIfEmpty.java:32)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableFlatMap$MergeSubscriber.onNext(FlowableFlatMap.java:163)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.subscriptions.DeferredScalarSubscription.complete(DeferredScalarSubscription.java:132)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.maybe.MaybeToFlowable$MaybeToFlowableSubscriber.onSuccess(MaybeToFlowable.java:70)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedMaybeObserver.lambda$onSuccess$1(RxInstrumentedMaybeObserver.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedMaybeObserver.onSuccess(RxInstrumentedMaybeObserver.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.maybe.MaybeDoOnEvent$DoOnEventMaybeObserver.onSuccess(MaybeDoOnEvent.java:86)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedMaybeObserver.lambda$onSuccess$1(RxInstrumentedMaybeObserver.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedMaybeObserver.onSuccess(RxInstrumentedMaybeObserver.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableElementAtMaybe$ElementAtSubscriber.onNext(FlowableElementAtMaybe.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableFlatMap$MergeSubscriber.tryEmit(FlowableFlatMap.java:282)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableFlatMap$InnerSubscriber.onNext(FlowableFlatMap.java:663)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchIfEmpty$SwitchIfEmptySubscriber.onNext(FlowableSwitchIfEmpty.java:59)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchIfEmpty$SwitchIfEmptySubscriber.onNext(FlowableSwitchIfEmpty.java:59)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableMap$MapSubscriber.onNext(FlowableMap.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap$SwitchMapSubscriber.drain(FlowableSwitchMap.java:307)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap$SwitchMapInnerSubscriber.onSubscribe(FlowableSwitchMap.java:366)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onSubscribe(InstrumentedSubscriber.java:75)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableJust.subscribeActual(FlowableJust.java:34)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedCallableFlowable.subscribeActual(RxInstrumentedCallableFlowable.java:65)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap$SwitchMapSubscriber.onNext(FlowableSwitchMap.java:129)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext$OnErrorNextSubscriber.onNext(FlowableOnErrorNext.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext$OnErrorNextSubscriber.onNext(FlowableOnErrorNext.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:52)
logs/tower.2020-04-05-1.log-	at io.micronaut.http.context.ServerRequestContext.lambda$instrument$0(ServerRequestContext.java:68)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext$OnErrorNextSubscriber.onNext(FlowableOnErrorNext.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.lambda$onNext$0(InstrumentedSubscriber.java:80)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onNext(InstrumentedSubscriber.java:84)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.subscriptions.ScalarSubscription.request(ScalarSubscription.java:55)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.subscriptions.SubscriptionArbiter.setSubscription(SubscriptionArbiter.java:99)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext$OnErrorNextSubscriber.onSubscribe(FlowableOnErrorNext.java:69)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.InstrumentedSubscriber.onSubscribe(InstrumentedSubscriber.java:75)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableScalarXMap.tryScalarXMapSubscribe(FlowableScalarXMap.java:91)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:46)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext.subscribeActual(FlowableOnErrorNext.java:40)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableScalarXMap.tryScalarXMapSubscribe(FlowableScalarXMap.java:93)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:46)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableScalarXMap.tryScalarXMapSubscribe(FlowableScalarXMap.java:93)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:46)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext.subscribeActual(FlowableOnErrorNext.java:40)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableScalarXMap.tryScalarXMapSubscribe(FlowableScalarXMap.java:93)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:46)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableScalarXMap.tryScalarXMapSubscribe(FlowableScalarXMap.java:93)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:46)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableOnErrorNext.subscribeActual(FlowableOnErrorNext.java:40)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableScalarXMap.tryScalarXMapSubscribe(FlowableScalarXMap.java:93)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:46)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableSwitchMap.subscribeActual(FlowableSwitchMap.java:49)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
logs/tower.2020-04-05-1.log-	at io.micronaut.reactive.rxjava2.RxInstrumentedFlowable.subscribeActual(RxInstrumentedFlowable.java:68)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.internal.operators.flowable.FlowableMap.subscribeActual(FlowableMap.java:37)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14918)
logs/tower.2020-04-05-1.log-	at io.reactivex.Flowable.subscribe(Flowable.java:14865)
```

**Comment by pditommaso** [comment]
Likely this is happening because 

```
TraceProgressRequest["tasks"]->java.util.ArrayList[0]->io.seqera.tower.domain.Task["tag"]
```

is receiving not a string value but a composed object.

**Comment by pditommaso** [comment]
Solved by c369d99b.
